hack/argo-cd/argocd-rbac-dev.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app.kubernetes.io/name: argocd-rbac-cm
+    app.kubernetes.io/part-of: argocd
+  name: argocd-rbac-cm
+  namespace: argocd
+data:
+  policy.csv: |
+    p, role:developer, applications, *, *, allow
```

Comment:
What's the purpose of having a separate account that has a very similar permissions as admin?

Reply:
This is an account for the developers and it only allows to handle applications

Reply:
Can you tell me if this is inline with what you are thinking?

1. Use a random password for the developer and admin account if the dev password flag is unset.
2. Use a static, known password for the developer and admin account if the dev password flag is set.

Reply:
Option 2 => Use a known password for the developer and admin account if the dev password flag is set

Reply:
Ok that sounds good to me. Looks like Gitea static password isn't working for some reason?

Reply:
> Ok that sounds good to me. Looks like Gitea static password isn't working for some reason?

For gitea when using dev-mode, we should still use as user: giteaAdmin and password = developer

Reply:
Since you are adding a user called developer to argocd, we may as well add the developer user in Gitea.